sky/utils/common_utils.py

```diff
@@ -61,11 +61,17 @@ def get_usage_run_id() -> str:
     return _usage_run_id
 
 
-def get_user_hash() -> str:
+def get_user_hash(force_fresh_hash: bool = False) -> str:
     """Returns a unique user-machine specific hash as a user id.
 
     We cache the user hash in a file to avoid potential user_name or
     hostname changes causing a new user hash to be generated.
+
+    Args:
+        force_fresh_hash: Bypasses the cached hash in USER_HASH_FILE and the
+            hash in the USER_ID_ENV_VAR and forces a fresh user-machine hash
+            to be generated.
```

Comment:
It might be good to specify the reason we need this argument in the comment. : )
```suggestion
            to be generated. This is only used for `kubernetes.ssh_key_secret_field_name` to avoid
            controllers sharing the same ssh key field name with the local laptop.
```